otx/algorithms/common/adapters/mmcv/tasks/stage.py

```diff
@@ -486,12 +487,34 @@ def get_model_classes(cfg):
     def get_model_ckpt(ckpt_path, new_path=None):
         """Return model ckpt from ckpt_path."""
         ckpt = CheckpointLoader.load_checkpoint(ckpt_path, map_location="cpu")
+        local_torch_hub_folder = torch.hub.get_dir()
         if "model" in ckpt:
             ckpt = ckpt["model"]
             if not new_path:
                 new_path = ckpt_path[:-3] + "converted.pth"
-            torch.save(ckpt, new_path)
+            if not torch.distributed.is_initialized() or dist.get_rank() == 0:
```

Comment:
This line actually prevents problem with multi-gpu training. If we remove this line - the checkpoint will be corrupted. Adding line using only 1 gpu with rank=0 will solve this issue, in case of single gpu training - first conditional won't let training fall with error